Datadog.Trace.sln

```diff
@@ -2496,20 +2496,20 @@ Global
 		{0F0F7D45-0E13-42B0-A158-8F303BBE8358}.Release|x64.ActiveCfg = Release|Any CPU
 		{0F0F7D45-0E13-42B0-A158-8F303BBE8358}.Release|x64.Build.0 = Release|Any CPU
 		{0F0F7D45-0E13-42B0-A158-8F303BBE8358}.Release|x86.ActiveCfg = Release|Any CPU
-		{BB2D2ED3-4378-4B44-974E-8BE80F0021EF}.Debug|Any CPU.ActiveCfg = Debug|x64
-		{BB2D2ED3-4378-4B44-974E-8BE80F0021EF}.Debug|Any CPU.Build.0 = Debug|x64
-		{BB2D2ED3-4378-4B44-974E-8BE80F0021EF}.Debug|x64.ActiveCfg = Debug|x64
-		{BB2D2ED3-4378-4B44-974E-8BE80F0021EF}.Debug|x64.Build.0 = Debug|x64
+		{BB2D2ED3-4378-4B44-974E-8BE80F0021EF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Why did this need to be changed? /cc @shurivich

Reply:
Without this change, I can't execute the Probe Tests locally in Release configuration, no matter what I do.

Reply:
@shurivich can you please review this change and provide feedback?